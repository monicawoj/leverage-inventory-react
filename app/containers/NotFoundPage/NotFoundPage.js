import React from 'react';
import { Link } from 'react-router-dom';

const NotFoundPage = () => {
  return (
    <section className="hero is-warning is-large is-centered has-text-centered">
      <div className="hero-body">
        <div className="container">
          <h1 className="title">
            Sorry, page not found!
          </h1>
        </div>
      </div>
    </section>
  );
};

export default NotFoundPage;
