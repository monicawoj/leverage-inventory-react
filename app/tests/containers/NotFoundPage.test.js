import React from 'react';
import { shallow } from 'enzyme';

import NotFoundPage from 'containers/NotFoundPage';

describe('<NotFoundPage />', () => {
  it('should shallow render without breaking', () => {
    const wrap = shallow(<NotFoundPage />);
    expect(wrap).toMatchSnapshot();
  });
});
