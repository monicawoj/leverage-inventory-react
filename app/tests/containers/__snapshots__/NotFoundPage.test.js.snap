// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should shallow render without breaking 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFoundPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="hero-body"
      >
        <div
          className="container"
        >
          <h1
            className="title"
          >
            Sorry, page not found!
          </h1>
          <h2
            className="subtitle"
          >
            Let's get back to the beer,
            <Link
              className="has-text-link"
              replace={false}
              to="/"
            >
               shall we?
            </Link>
          </h2>
        </div>
      </div>,
      "className": "hero is-warning is-large is-centered has-text-centered",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container"
        >
          <h1
            className="title"
          >
            Sorry, page not found!
          </h1>
          <h2
            className="subtitle"
          >
            Let's get back to the beer,
            <Link
              className="has-text-link"
              replace={false}
              to="/"
            >
               shall we?
            </Link>
          </h2>
        </div>,
        "className": "hero-body",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1
              className="title"
            >
              Sorry, page not found!
            </h1>,
            <h2
              className="subtitle"
            >
              Let's get back to the beer,
              <Link
                className="has-text-link"
                replace={false}
                to="/"
              >
                 shall we?
              </Link>
            </h2>,
          ],
          "className": "container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Sorry, page not found!",
              "className": "title",
            },
            "ref": null,
            "rendered": "Sorry, page not found!",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Let's get back to the beer,",
                <Link
                  className="has-text-link"
                  replace={false}
                  to="/"
                >
                   shall we?
                </Link>,
              ],
              "className": "subtitle",
            },
            "ref": null,
            "rendered": Array [
              "Let's get back to the beer,",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": " shall we?",
                  "className": "has-text-link",
                  "replace": false,
                  "to": "/",
                },
                "ref": null,
                "rendered": " shall we?",
                "type": [Function],
              },
            ],
            "type": "h2",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="hero-body"
        >
          <div
            className="container"
          >
            <h1
              className="title"
            >
              Sorry, page not found!
            </h1>
            <h2
              className="subtitle"
            >
              Let's get back to the beer,
              <Link
                className="has-text-link"
                replace={false}
                to="/"
              >
                 shall we?
              </Link>
            </h2>
          </div>
        </div>,
        "className": "hero is-warning is-large is-centered has-text-centered",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="container"
          >
            <h1
              className="title"
            >
              Sorry, page not found!
            </h1>
            <h2
              className="subtitle"
            >
              Let's get back to the beer,
              <Link
                className="has-text-link"
                replace={false}
                to="/"
              >
                 shall we?
              </Link>
            </h2>
          </div>,
          "className": "hero-body",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h1
                className="title"
              >
                Sorry, page not found!
              </h1>,
              <h2
                className="subtitle"
              >
                Let's get back to the beer,
                <Link
                  className="has-text-link"
                  replace={false}
                  to="/"
                >
                   shall we?
                </Link>
              </h2>,
            ],
            "className": "container",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Sorry, page not found!",
                "className": "title",
              },
              "ref": null,
              "rendered": "Sorry, page not found!",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Let's get back to the beer,",
                  <Link
                    className="has-text-link"
                    replace={false}
                    to="/"
                  >
                     shall we?
                  </Link>,
                ],
                "className": "subtitle",
              },
              "ref": null,
              "rendered": Array [
                "Let's get back to the beer,",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": " shall we?",
                    "className": "has-text-link",
                    "replace": false,
                    "to": "/",
                  },
                  "ref": null,
                  "rendered": " shall we?",
                  "type": [Function],
                },
              ],
              "type": "h2",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
